docs/UserGuide.adoc

```diff
@@ -111,6 +111,29 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+== Deleting multiple people : `deleteM`
+
+Deletes the specified people from the address book. Irreversible. +
+Format: `deleteM INDEX, INDEX, INDEX`
+****
+Deletes the people at the specified `INDEX`.
```

Comment:
Can add a 's' behind `INDEX` i.e. `INDEX`s to make it clearer. Or `INDEX`(s) to show that it can also accept only one argument

Reply:
Noted. Thank you! I will update in the level3/4.